.github/workflows/ci-development.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     uses: ./.github/workflows/upgrade-test.yml
     secrets: inherit
     with:
-      run-job: false
+      run-job: true
```

Comment:
This needs to be removed before merging I guess?

Reply:
yep just left for now in case some PR comments come in that warrant changes